lib/objectLog.ts

```diff
@@ -185,6 +185,7 @@ export class ResourceTable implements DataTable {
         e.resourceType,
         e.resourceId,
         e.resourceState,
+        (e?.metadata?.actionNames || []).join(','),
```

Comment:
CSV array within a CSV row. Either
* add a header per action
* use a different delimiter.
* Update `ObjectLogCsv.writeCsv` to escape or quote values with a comma in them (it's dumb atm)

Reply:
Updated to `|`